en/identity-server/next/docs/apis/restapis/organization-user-share.yaml

```
@@ -317,7 +324,7 @@ paths:
         - User Accessible Organizations
       summary: Get organizations a user has access to
       description: |
-        This API retrieves the list of organizations where the specified user has shared access, with support for pagination and filtering.
+        This API retrieves the list of organizations where the specified user has shared access. Supports pagination and filtering. 
```

```suggestion
        This API retrieves the list of organizations where the specified user has shared access.
```
Let's remove pagination support.

---

Verified.